docs/src/development/macos.md

```diff
@@ -41,6 +41,7 @@ If you are developing collaborative features of Zed, you'll need to install the
   ```sh
   brew install livekit foreman
   ```
+- Follow the steps in the [collab README](../../../crates/collab/README.md) to configure the Postgres database for integration tests
```

Comment:
We can't use relative links like this in the docs, as they will break when deployed to zed.dev/docs.

This needs to be an absolute URL to GitHub.

Reply:
Should be fixed